src/test/MsQuicTests.h

```diff
@@ -9,7 +9,9 @@
 
 --*/
 
+#ifndef QUIC_OFFICIAL_RELEASE
 #define QUIC_API_ENABLE_PREVIEW_FEATURES
+#endif
```

Comment:
Somehow on Linux, it seems `QUIC_API_ENABLE_PREVIEW_FEATURES` is not getting defined, but AFAICT `QUIC_OFFICIAL_RELEASE` is not defined. I cannot figure out why.

Reply:
Is this all `QUIC_OFFICIAL_RELEASE` is used for?  I want to make sure we don't change behavior with this flag, since that'll potentially moot testing.

Reply:
That's all it's doing now.